composer.json

```diff
@@ -10,7 +10,7 @@
     },
     "require": {
         "php": ">=7.1",
-        "jane-php/jane-php": "dev-feature.php83-v2"
+        "jane-php/jane-php": "dev-nocworx-php83"
```

Comment:
alias this so we don't have issues with minimum-stability.
```json
  "jane-php/jane-php": "dev-nocworx-php83 as 7.5.10083"
```

Reply:
Changed to the answer of life, `7.5.10042`